### ISSUE TITLE
Close the Gitter channel 👋

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5#2037]
**tl;dr:**

* CKEditor 5's Gitter channel stopped fulfilling its role, so we made a decision to close it.
* Use this repository, [StackOverflow](https://stackoverflow.com/tags/ckeditor5), https://twitter.com/ckeditor, or https://ckeditor.com/contact to contact us and the CKEditor 5 community. 

---

We opened the [Gitter channel](https://gitter.im/ckeditor/ckeditor5) in order to serve a specific purpose – allow quick communication in simpler matters. We knew that it must not duplicate the activities that should normally happen on GitHub and StackOverflow. But there's a certain sort of questions that don't fit StackOverflow and for which GitHub seems to be an overkill.

To strengthen our point, we wrote down a guide when Gitter can be used:

> CKEditor 5 Gitter channel is the place where you can discuss all matters that do not fit GitHub or Stack Overflow.
>
> Gitter is convenient for asking quick questions, however, it is not the best place for more complex topics which might interest other developers (Gitter logs are indexed via Google but rather poorly). Therefore, try to avoid lengthy discussions on topics which would make sense to be reported either on GitHub or Stack Overflow.
>
> DOs:
>
> * Is it a known bug that “…”?
> * Do you plan to tag the new release this week?
> * I can’t find XYZ’s documentation. Is it available somewhere?
> * I’m in Warsaw. Anyone for a coffee?
>
> DON’Ts:
>
> * How to extend the selection to the end of the block?
> * Could you implement support for XYZ?
> * I can’t make X and Y work together in my app. I developed this plugin and…
> * The editor crashes if I press Enter. Please help.
> * And most importantly, be kind to everyone. 

This worked for a couple of years. The CKEditor 5 community was initially quite small and we were able to handle legit questions as well as forward other kinds of messages to other channels.

Unfortunately, with time, the quality of Gitter's channel dropped. The community grew, which means that the overall number of messages grew too. And since the fraction of them that should be posted elsewhere is rather constant, their total amount became impossible for us to handle. 

Therefore, we made a decision to close the Gitter channel. It does not make sense for this channel to exist if 90% of messages are left unanswered. Right now, it is a false promise of a place where you can get help. And we feel that it's better to not make promises that we cannot keep.

With Gitter being closed all the technical communication with the team should happen through this repository. In urgent cases (e.g. broken CDN) you can also contact us via https://twitter.com/ckeditor or https://ckeditor.com/contact. Via those channels, we are able to better moderate the feedback and help whenever we can. Also, GitHub's SEO is far greater than that of Gitter which helps even with simple topics if more people stumble upon a certain problem.

We'd like to thank everyone who joined us on Gitter. Especially in the early phases of the project, when we needed your feedback the most :) Thank you, everyone ❤️

**Comment by robclancy** [comment]
Is there not a discord? It's hard to contribute to a project of this scale when you can't even discuss things in real time.

**Comment by Reinmar** [comment]
Live communication on gitter was never a thing. We are not able to monitor it throughout the entire workday as we have our own work to do. And then the timezone difference kicks in (we're all Europe-based) and there's no chance that this could work.

Async communication via GitHub (or other similar means, but we just chose to have everything in one place) is the best solution which ensures:

* no one feels that we should be replying within minutes,
* communication is threaded,
* logs from the communication are searchable both via Google and additional filters (labels, milestones),
* we have some other features like the ability to mute or close a thread, subscribe/unsubscribe, etc.

We'd love to be able to work in close cooperation with everyone out there, but the reality is that in case of a project like CKEditor 5 it just don't work. And we'd rather have core devs contributing to the project. 

That said – while GitHub is always a good start, if there's really a need for a live discussion, there's no problem to arrange it.

**Comment by robclancy** [comment]
I don't really agree with this. A discord doesn't mean you have to reply instantly or even join it. And github being used for questions just makes things messy. Anyway... doesn't matter.

FYI gitter is still showing here https://ckeditor.com/ckeditor-5/resources/.

**Comment by Reinmar** [comment]
> FYI gitter is still showing here https://ckeditor.com/ckeditor-5/resources/.

Thanks. I didn't even know that it was listed there :D

> And github being used for questions just makes things messy.

Why do you think it's messy? Personally, I cannot really imagine the opposite – having some questions and discussions split to another place. The reason for that is that people usually don't know where to start a thread. Do they have a question? Or report a bug? Will this discussion turn into A or into B? Some questions that should be on Discord will be asked on GH anyway. Some bugs would be reported there. Actually – the same happened with Gitter. It was meant to be a completely different place, but with people ignoring or not even reading the Dos/Donts, 90% of what happened there was forwarding people to other places. Perhaps on Discord, that'd be better, but still... At the end of the day, I'm not sure I'd actually know why we have it.
